main.rb

```diff
@@ -0,0 +1,54 @@
+require './app'
+
+def main
+  books = []
+  student = []
+  people = []
+  rental = []
+  option(books, student, people, rental)
+end
+
+def start_up
+  puts 'Please choose an option by entering a number:'
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Ceate a new book'
+  puts '5 - Ceate a new rental'
+  puts '6 - List all rental for given ID'
+  puts '7 - Exit'
+end
+
+def grab_choice(choice, books, people, rental)
+  case choice
+  when '1'
+    puts 'list of all books :'
+    display_list_of(books)
+  when '2'
+    puts 'List of all people:'
+    display_list_of_people(people)
+  when '3'
+    add_new_person(people)
```

Comment:
Choosing option 3 for creating either a Student or Teacher throws an error.

![image](https://user-images.githubusercontent.com/38088818/181908057-cdcf4a38-b2ae-476d-8806-aaae823b954e.png)

- Kindly look into this and make sure that a user can successfully create a person of choice.

Note: This is an `(ArguementError)`  which means that you are providing the `grab_choice` method with the wrong number of arguments.